src/test/scala/org/openeo/opensearch/CreoFeatureCollectionTest.scala

```diff
@@ -21,16 +22,20 @@ object CreoFeatureCollectionTest {
     Arguments.of(Some("30*F*"), Boolean.box(false)),
     Arguments.of(Some("3*U**"), Boolean.box(true)),
```

Comment:
"*" does the same as "**" is that intentional?

Reply:
I would expect them to do the same yes. I would not expect the user to actually pass "**" though.